core/server/services/permissions/providers.js

```diff
@@ -5,7 +5,7 @@ var _ = require('lodash'),
 
 module.exports = {
     user: function (id) {
-        return models.User.findOne({id: id, status: 'all'}, {withRelated: ['permissions', 'roles', 'roles.permissions']})
+        return models.User.findOne({id: id, status: 'active'}, {withRelated: ['permissions', 'roles', 'roles.permissions']})
```

Comment:
Can this go in as a separate commit/PR? It's a different issue to the session invalidation on password change

Reply:
Yup this whole thing will be split into just 2 commits!